pkg/cc/config/topaz_config.go

```diff
@@ -56,6 +61,9 @@ func defaults(v *viper.Viper) {
 }
 
 func (c *Config) validation() error {
+	if c.Version != 1 {
```

Comment:
Should the config version be a public constant, or should there be a public func config.IsSupportedVersion(v int) bool, or maybe both as I can imaging the constant is needed/used in the template where the check is used elswehere ?

Reply:
set a constant `ConfigFileVersion` in topaz config and used in configuration template